include/vcpkg/base/messages.h

```diff
@@ -741,6 +769,10 @@ namespace vcpkg
                     "",
                     "Visual Studio Code was not found and the environment variable {env_var} is not set or invalid.");
     DECLARE_MESSAGE(ErrorVsCodeNotFoundPathExamined, (), "", "The following paths were examined:");
+    DECLARE_MESSAGE(ErrorWhileParsing,
+                    (msg::value),
+                    "'{value}' is the file name.",
```

Comment:
If it's the file name, should it not be `msg::path`? Alternately this should just say `vcpkg-configuration.json` directly in the message text.